arch/risc-v/include/arch.h

```diff
@@ -1,32 +1,20 @@
 /****************************************************************************
  * arch/risc-v/include/arch.h
  *
- *   Copyright (C) 2016 Ken Pettit. All rights reserved.
```

Comment:
it's better to move the copyright change to another PR to simplify the review.

Reply:
I just used the one from rv64 which was Apache but this file I am replacing is BSD which is why it looks like I license change.